ztools/ztls/ztls_names.go

```diff
@@ -435,6 +436,21 @@ func init() {
 	pointFormatNames[0] = "uncompressed"
 	pointFormatNames[1] = "ansiX962_compressed_prime"
 	pointFormatNames[2] = "ansiX962_compressed_char2"
+
+	// https://tools.ietf.org/html/draft-ietf-tls-tls13-18#section-4.2.3
```

Comment:
Currently there is only a pointFormatUncompressed defined as a uint8 constant in common.go. Do you mean just add constants for the other uint8 values and then change the key to the constant instead of the value?